### PR TITLE
Update ALCOR-0.9.4

### PR DESCRIPTION
**Pull request by politas**
Advised this release works fine in KSP 1.0.5

### CODE REVIEW

**Comment by politas** [comment]
#rebuild

File: ALCOR/ALCOR-0.9.4.ckan [hunk 1]
```diff
@@ -6,12 +6,11 @@
...
     "resources": {
-        "homepage": "http://forum.kerbalspaceprogram.com/threads/54925-1-0-2-ALCOR-Advanced-Landing-Capsule-for-Orbital-Rendezvous-by-ASET-(05-05-2015)",
-        "kerbalstuff": "https://kerbalstuff.com/mod/1015/ALCOR",
-        "x_screenshot": "https://kerbalstuff.com/content/alexustas_471/ALCOR/ALCOR-1437374156.0992417.jpg"
+        "homepage": "http://forum.kerbalspaceprogram.com/threads/54925-1-0-2-ALCOR-Advanced-Landing-Capsule-for-Orbital-Rendezvous-by-ASET-(05-05-2015)"
     },
     "version": "0.9.4",
-    "ksp_version": "1.0.4",
+    "ksp_version_min": "1.0.4",
+    "ksp_version_max": "1.0.4",
```
Comment (plague006) [inline ALCOR/ALCOR-0.9.4.ckan]: .

Comment (politas) [inline ALCOR/ALCOR-0.9.4.ckan]: Yes, I know.
Trying to work out the failed dependency

**Comment by politas** [comment]
Ok, Jenkins can't seem to work out the dependency, but I think we should be right to merge this.

**Comment by plague006** [comment]
Tests fine locally when using ksp version 1.0.4.

**Comment by techman83** [comment]
Passes now that it's merged. This is where the cli -c bug bites us occasionally. If I knew more C# I'd fix it. CKAN doesn't actually install the CKAN file on the CLI, it installs whatever is the latest in the metadata. I thought we injected everything to get around that but we don't.

**Comment by techman83** [comment]
Relevant Bug: KSP-CKAN/CKAN#1356

**Comment by politas** [comment]
I thought I'd hit this same behaviour before.

**Pull request merged**